store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -645,7 +648,8 @@ func (mvcc *MVCCLevelDB) Prewrite(req *kvrpcpb.PrewriteRequest) []error {
 	return errs
 }
 
-func checkConflictValue(iter *Iterator, m *kvrpcpb.Mutation, startTS uint64) error {
+// callerTS is startTS for optimistic transactions, and forUpdateTS for pessimistic transactions
+func checkConflictValue(iter *Iterator, m *kvrpcpb.Mutation, callerTS uint64, startTS uint64) error {
```

Comment:
The name `callerTS` may be confused with `callerStartTS` used in `CheckTxnStatus`.

Reply:
Or just use `forUpdateTS` ?

Reply:
OK